Model/Payment/AbstractMethod.php

```diff
@@ -446,4 +447,4 @@ protected function isValidStatus($bill)
 
         return in_array($bill['status'], $billStatus) || $chargeStatus;
     }
-}
+}
```

Comment:
💅🏼 Para mantermos o padrão de termos uma linha em branco no final, o que acha?
```suggestion
}

```